test/FunctionalTests/Microsoft.OData.Core.Tests/ScenarioTests/UriParser/ExpandAndSelectPathExtractingTests.cs

```diff
@@ -228,16 +228,14 @@ public void ExpandNavigationThatIsNotSelectedAutomaticallySelectsLink()
         {
             const string selectClauseText = "Navigation2";
             const string expandClauseText = "Navigation1";
-            const string expectedSelectClause = "Navigation2,Navigation1";
-            this.ParseAndExtract(selectClauseText: selectClauseText, expandClauseText: expandClauseText, expectedSelectClauseFromOM: expectedSelectClause, expectedExpandClauseFromOM: "Navigation1");
+            this.ParseAndExtract(selectClauseText: selectClauseText, expandClauseText: expandClauseText, expectedSelectClauseFromOM: selectClauseText, expectedExpandClauseFromOM: "Navigation1");

```

Comment:
>"Navigation1" [](start = 179, length = 13)

SHould just be expandClauseText

Reply:
Yup. Updated.

---
In reply to: [226132107](https://github.com/OData/odata.net/pull/1286#discussion_r226132107) [](ancestors = 226132107)